test/NuGet.Tests.Apex/NuGet.Tests.Apex.Daily/NuGetConsoleTestCase.cs

```diff
@@ -96,17 +96,18 @@ public async Task UpdateAllPackagesInPMC(ProjectTemplate projectTemplate, string
         }
 
         [DataTestMethod]
-        [DynamicData(nameof(GetIOSTemplates), DynamicDataSourceType.Method)]
+        [DynamicData(nameof(GetMauiTemplates), DynamicDataSourceType.Method)]
         [Timeout(DefaultTimeout)]
-        public async Task InstallPackageForIOSProjectInPMC(ProjectTemplate projectTemplate)
+        public async Task InstallPackageForMauiProjectInPMC(ProjectTemplate projectTemplate)
         {
             EnsureVisualStudioHost();
             using (var simpleTestPathContext = new SimpleTestPathContext())
             {
-                // Arrange
-                var packageName = "IOSTestPackage";
+                // Arrange
+                var packageName = "IOSTestPackage";
                 var v100 = "1.0.0";
                 await CommonUtility.CreatePackageInSourceAsync(simpleTestPathContext.PackageSource, packageName, v100);
+                simpleTestPathContext.Settings.AddSource(NuGetConstants.NuGetHostName, NuGetConstants.V3FeedUrl);
```

Comment:
Just a thought that I haven't tried, but looks feasible, is instead of adding a remote feed (nuget.org), could the necessary packages just be created in the simpleTestPathContext?

For example, 
`await CommonUtility.CreatePackageInSourceAsync(simpleTestPathContext.PackageSource, "Xamarin.Android.AndroidX.Common", v100);`

If this works, it seems better to me. The test's goal is that the console opens and installs a package, that package is in the Output pane, and no errors exist. It doesn't matter whether we download anything from a live feed or not, or even whether it's a real package with any assemblies.

Reply:
That'd be a constantly moving target. The auto referenced packages change.

Reply:
I see, so we want to test the side effect of this test that the real MAUI template can be downloaded from nuget.org.